packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -202,6 +202,7 @@ export type CallAdapterClientState = {
     alternateCallerId?: string;
     environmentInfo?: EnvironmentInfo;
     roleHint?: Role;
+    isCameraOn?: boolean;
```

Comment:
Just thinking of this for the future a bit - Is it possible that camera has more than one state? like "on", "pending" or we wanna expose more property from camera?

Would be nice if we could have a future proof name like: 
cameraStatus: 'on' | 'off'

Because of this name, we can make it like object or add more string as options to make it flexible

Reply:
Or even more - we might wanna expose microphoneStatus as well - do we wanna merge them together as 
```
devicesStatus: {
microphone:
camera:
}
```

Maybe cameraStatus is good already for us